converter/src/main/resources/metamorph.xsd

```diff
@@ -749,6 +750,28 @@
 			</attribute>
 		</complexType>
 	</element>
+	
```

Comment:
I would propose to extend the metamorph.xsd in our metafacture fork, otherwise we may run in conflicts with this metamorph.xsd there.